.github/workflows/release.yml

```
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        TARGET: ["ubuntu:20.04", "ubuntu:22.04", "debian:11", "debian:12"]
+        TARGET: ["ubuntu:22.04", "ubuntu:24.04", "debian:11", "debian:12"]
```

```suggestion
        TARGET: ["ubuntu:22.04", "ubuntu:24.04", "debian:12"]
```

We loose support for Debian 11 as its Eigen version is still 3.3.9 and we now require 3.4.0